_redirects.ejs

```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```

Comment:
Seems better to do:

```
/    /index.html
/*  / 307
```

That way we have a canonical URL for the app.

Reply:
this is the default behavior IIRC.